tubemq-server/src/main/java/org/apache/tubemq/server/broker/metadata/TopicMetadata.java

```diff
@@ -37,10 +37,8 @@
     private int unflushThreshold = 1000;
```

Comment:
Can you revert the history changes, then rebase codes, and then submit the final changes again? The historical changes affect the comparison of the changes.